Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/FitGaussianPeaks.py

```diff
@@ -285,6 +299,9 @@ def estimate_single_parameters(self, xvals, yvals, centre, win_size):
             rside = 5 + lside
         x_range = xvals[lside:rside]
         y_range = yvals[lside:rside]
+        if len(x_range) < 5:
+            self.log().warning(f"Peak at {xvals[centre]} cannot be fitted")
```

Comment:
Could we add a reason to the warning string to indicate to the user why the peak cannot be fitted? It may stop them thinking this is a bug :)